avcodec.cpp

```diff
@@ -300,6 +300,9 @@ static int avcodec_decoder_copy_frame(const avcodec_decoder d, opencv_mat mat, A
                                                 NULL,
                                                 NULL,
                                                 NULL);
+        const int *src_coeffs = sws_getCoefficients(frame->colorspace);
+        const int *dst_coeffs = sws_getCoefficients(0);
```

Comment:
I wonder if it would make more sense to specify `SWS_CS_ITU709` (which evals to 1) here rather than 0 (which has no `#define` but appears to use the same coefficients as `SWS_CS_ITU709`).